afs-cassandra/src/test/java/com/powsybl/afs/cassandra/CassandraDataSplitTest.java

```diff
@@ -66,5 +73,27 @@ public void test() throws IOException {
         assertTrue(storage.removeData(nodeInfo.getId(), "a"));
         assertTrue(storage.getDataNames(nodeInfo.getId()).isEmpty());
         assertFalse(storage.readBinaryData(nodeInfo.getId(), "a").isPresent());
+
+        try (OutputStream os = storage.writeBinaryData(nodeInfo.getId(), "a")) {
+            byte[] bytes = "aaaaaaaaaabbbbbbbbbbccccccccccdddddddddd".getBytes(StandardCharsets.UTF_8);
+            os.write(bytes, 0, 5);
+            os.write(bytes, 5, 10);
+            os.write(bytes, 15, 4);
+            os.write(bytes, 19, 3);
+            os.write(bytes, 22, 18);
+        }
+        storage.flush();
+
+        InputStream is3 = storage.readBinaryData(nodeInfo.getId(), "a").orElse(null);
+        assertNotNull(is3);
+        assertEquals("aaaaaaaaaabbbbbbbbbbccccccccccdddddddddd", new String(ByteStreams.toByteArray(is3), StandardCharsets.UTF_8));
+        is3.close();
+
+        ResultSet resultSet = cassandraTestContext.getSession().execute(select(CHUNKS_COUNT).from(NODE_DATA)
+                .where(eq(ID, UUID.fromString(nodeInfo.getId())))
+                .and(eq(NAME, "a")));
+        Row firstRow = resultSet.one();
+        assertNotNull(firstRow);
+        assertEquals(new Integer(4), firstRow.get(0, Integer.class));
```

Comment:
This is uggly... Why allocating a new Integer like this? Is the autoboxing doesn't work it that specific case? If not, use valueOf() instead